src/AspNetCoreModuleV2/CommonLib/LoggingHelpers.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+
+IOutputManager*
+LoggingHelpers::CreateLoggingProvider(
+    bool fIsLoggingEnabled,
+    bool fIsConsoleWindow,
```

Comment:
this variable name is misleading as you use it to determine if to create NullConsoleManager. You can add comments for better code maintenence